backend/maelstro/__init__.py

```diff
@@ -0,0 +1,62 @@
+"""
+Entry point scripts for maelstro backend server
+"""
+
+import uvicorn
+from fastapi_cli.utils.cli import get_uvicorn_log_config
+
+
+def dev():
```

Comment:
Not sure we really need those different entrypoints.

I would consider using environment variables or overwriting command in docker-compose.override.yaml

Note that there might already exists such env vars in uvicorn

Reply:
it's comfortable for now, we can discuss this later